Index: gp-includes/things/translation.php
===================================================================
--- gp-includes/things/translation.php	(revision 1039)
+++ gp-includes/things/translation.php	(working copy)
@@ -103,8 +103,11 @@
 			$where[] = '(' . implode( ' OR ', array_map( function( $x ) use ( $like ) { return "($x $like)"; }, array( 'o.singular', 't.translation_0', 'o.plural', 't.translation_1', 'o.context', 'o.references' ) ) ) . ')';
 		}
 		if ( gp_array_get( $filters, 'before_date_added' ) ) {
-			$where[] = $gpdb->prepare( 't.date_added > %s', gp_array_get( $filters, 'before_date_added' ) );
+			$where[] = $gpdb->prepare( 't.date_added >= %s', gp_array_get( $filters, 'before_date_added' ) );
 		}
+		if ( gp_array_get( $filters, 'after_date_added' ) ) {
+			$where[] = $gpdb->prepare( 't.date_added <= %s', gp_array_get( $filters, 'after_date_added' ) );
+		}
 		if ( gp_array_get( $filters, 'translation_id' ) ) {
 			$where[] = $gpdb->prepare( 't.id = %d', gp_array_get( $filters, 'translation_id' ) );
 		}
